src/Analyzer.Utilities/CSharpSyntaxGeneratorExtensions.cs

```diff
@@ -1,22 +1,23 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System.Collections.Generic;
-using System.Composition;
-using System.Diagnostics;
+﻿using System.Diagnostics;
```

Comment:
Likewise, I removed CSharpOverrideMethodsOnComparableTypes.Fixer.cs and BasicOverrideMethodsOnComparableTypes.Fixer.vb, and the base class OverrideMethodsOnComparableTypes.Fixer.cs now uses the extension methods. But Git thinks I renamed CSharpOverrideMethodsOnComparableTypes.Fixer.cs to CSharpSyntaxGeneratorExtensions.cs.